web/cobrands/sass/_mixins.scss

```diff
@@ -78,6 +78,7 @@ $button-margin-left: 0 !default;
     &:focus{
       color: $focus-text !important;
       background: $focus-bg-bottom;
+      outline: $focus-outline-width solid $focus-outline-color;
```

Comment:
This doesn't use the $focus-outline-style passed in, should it be?

Reply:
Sorry, didn't answer your question about outline - I don't know, the focus colour does seem like it might be enough? I can't see outline used on e.g. gov.uk like this.